src/androidTest/java/cash/z/ecc/android/sdk/tool/SaplingParamToolTest.kt

```diff
@@ -0,0 +1,19 @@
+package cash.z.ecc.android.sdk.tool
+
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import junit.framework.Assert.assertEquals
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+class SaplingParamToolTest {
+
+    @Test
+    fun validateParamTest(){
+        val pathDir = "pathParamsDir"
```

Comment:
This is fine for now, we can flesh this test out more as part of https://github.com/zcash/zcash-android-wallet-sdk/issues/202